index.js

```diff
@@ -3,7 +3,11 @@
 const copyProperty = (to, from, property, ignoreNonConfigurable) => {
 	// `Function#length` should reflect the parameters of `to` not `from` since we keep its body.
 	// `Function#prototype` is non-writable and non-configurable so can never be modified.
-	if (property === 'length' || property === 'prototype') {
+	// `Function#arguments` and `Function#caller` are present in Reflect.ownKeys for some devices in react-native
+	if (property === 'length' ||
```

Comment:
The React Native workaround should be a separate if statement and not be mixed with the "correct" code.